Tests/L0/Common-VstsAzureHelpers_/_suite.ts

```diff
@@ -82,6 +82,9 @@ describe('Common-VstsAzureHelpers_ Suite', function () {
         it('(Initialize-AzureSubscription) throws when RM and cert auth', (done) => {
             psr.run(path.join(__dirname, 'Initialize-AzureSubscription.ThrowsWhenRMAndCertAuth.ps1'), done);
         })
+        it('(Initialize-AzureSubscription) passes environment value to Initialize-AzureSubscription , if null default environment azurecloud', (done) => {
```

Comment:
This description is messed up. The test doesn't assert that values are passed to Initialize-AzureSubscription. Following the convention in the file, and convention of the file name, shouldn't this description be:
`(Initialize-AzureSubscription) passes values when cert auth and environment`?

The descriptions print out when gulp runs, not the file names. It's nice for the descriptions to correlate directly to the files without having to reference suite.js as an index.